packages/dso-toolkit/src/components/header/header.stories-of.ts

```diff
@@ -111,6 +113,20 @@ export function storiesOfHeader<Implementation, Templates, TemplateFnReturnType>
       },
     });
 
+    stories.add("with link to help", template, {
+      args: {
+        showHelp: true,
+        helpUrl: "#help",
+      },
+    });
+
+    stories.add("with button to help", template, {
+      args: {
+        showHelp: true,
+        helpUrl: "",
```

Comment:
Moeten we dan een lege `helpUrl` zetten? Of kunnen we deze arg ook achterwege laten?

Reply:
Ja, die mag weg!